features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml

```diff
@@ -325,11 +614,49 @@
                 <Scope name="internal_group_mgt_view"/>
```

Comment:
Shouldn't we remove the newly added scopes after IS 7.0 release from this legacy scope collection?